scripts/icd_dispatch_generated.c.mako

```diff
@@ -78,14 +78,23 @@ ${("", "static ")[disp]}CL_API_ENTRY ${api.RetType} CL_API_CALL ${api.Name + (""
 %if disp == 0:
 #if defined(CL_ENABLE_LAYERS)
     if (khrFirstLayer)
+%  if api.Name == "clSVMFree":
```

Comment:
This is fine for now, but if we wanted to make this a little more general we could check for a `void` return type here vs. `clSVMFree` specifically.

Reply:
Yes it occurred to me, but I would have had to make this change several times throughout the file, so I kept it uniform. We will need to refactor it at some point.